monkey/monkey_island/cc/ui/src/styles/components/PluginSelectorTemplate.scss

```diff
@@ -0,0 +1,3 @@
+.active-checkbox {
+  background-color: $gray-300;
+}
```

Comment:
This style should go in `monkey_island/cc/ui/src/styles/components/AdvancedMultiSelect.scss`.